### PR TITLE
Add CLI for monitoring transformation status

### PR DESCRIPTION
**Pull request by chcn**
Example usage:
```
(base) ➜  alluxio git:(transform-status-cli) ✗ ./bin/alluxio table transformStatus
database: default
table: test
transformation: write(hive).option(hive.num.files, 2)
job ID: 1571960683001
job status: RUNNING

(base) ➜  alluxio git:(transform-status-cli) ✗ ./bin/alluxio table transformStatus 1571960683001
database: default
table: test
transformation: write(hive).option(hive.num.files, 2)
job ID: 1571960683001
job status: COMPLETED

(base) ➜  alluxio git:(transform-status-cli) ✗ ./bin/alluxio table transformStatus 1571960683001
database: default
table: test
transformation: write(hive).option(hive.num.files, 1)
job ID: 1571960683002
job status: FAILED
job error: [error message from job service]
```

### CODE REVIEW

**Comment by chcn** [comment]
@yuzhu updated the PR, PTAL

File: table/server/master/src/main/java/alluxio/master/table/transform/TransformManager.java [hunk 1]
```diff
@@ -249,55 +251,60 @@ private void completePartitionTransformation(String dbName, String tableName, Tr
...
+        job.setJobStatus(Status.COMPLETED);
       } catch (IOException e) {
-        LOG.error("Failed to update partition layouts for database {} table {}",
-            table.getFirst(), table.getSecond());
+        String error = String.format("Failed to update partition layouts for database %s table %s",
+            job.getDb(), job.getTable());
+        LOG.error(error);
+        job.setJobStatus(Status.FAILED);
+        job.setJobErrorMessage(error);
       }
```
Comment (yuzhu) [inline table/server/master/src/main/java/alluxio/master/table/transform/TransformManager.java]: would there be a problem if the server crashes between the partition update is journaled and the job completion is journaled?

File: table/server/master/src/main/java/alluxio/master/table/transform/TransformManager.java [hunk 2]
```diff
@@ -231,14 +204,41 @@ public long execute(String dbName, String tableName, TransformDefinition definit
...
   /**
    * Periodically polls the job service to monitor the status of the running transformation jobs,
    * if a transformation job succeeds, then update the Table partitions' layouts.
    */
   private final class JobMonitor implements HeartbeatExecutor {
-    private void completePartitionTransformation(String dbName, String tableName, TransformJob job)
-        throws IOException {
-      Table table = mCatalog.getTable(dbName, tableName);
+    private void updatePartition(TransformJobInfo job) throws IOException {
+      Table table = mCatalog.getTable(job.getDb(), job.getTable());
```
Comment (yuzhu) [inline table/server/master/src/main/java/alluxio/master/table/transform/TransformManager.java]: could you move this update to inside `mCatalog`? locking is done in Catalog right now, and this is accessing unlocked Table object.  The future plan is to make mCatalog only return TableView object and DbView objects. All updates has to go through catalog.

**Comment by yuzhu** [comment]
@chcn also checkstyle
```
[INFO] Starting audit...
/usr/src/alluxio/table/server/master/src/main/java/alluxio/master/table/transform/TransformManager.java:88: error: Line is longer than 100 characters (found 103).
/usr/src/alluxio/table/server/master/src/main/java/alluxio/master/table/transform/TransformJobInfo.java:48: error: Line is longer than 100 characters (found 108).
Audit done.```

File: table/server/master/src/main/java/alluxio/master/table/transform/TransformManager.java (hunk 1 above)
Comment (chcn) [reply table/server/master/src/main/java/alluxio/master/table/transform/TransformManager.java]: resolved offline

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**